### PR TITLE
Pre-requisite packages (Bootstrap and Angular material)

### PR DESCRIPTION
**Pull request by Nathanael-Rayapin**

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -3,10 +3,11 @@ import { BrowserModule } from '@angular/platform-browser';
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
 import { LayoutModule } from './page/layout/layout.module';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 
 @NgModule({
   declarations: [AppComponent],
-  imports: [BrowserModule, AppRoutingModule, LayoutModule],
+  imports: [BrowserModule, AppRoutingModule, LayoutModule, BrowserAnimationsModule],
```
Comment (etsraphael) [inline src/app/app.module.ts]: A material module should be included here to import all the modules of Angular Material. We can do it together

File: src/index.html [hunk 2]
```diff
@@ -6,7 +6,10 @@
     <base href="/" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link rel="icon" type="image/x-icon" href="favicon.ico" />
-  </head>
+      <link rel="preconnect" href="https://fonts.gstatic.com">
```
Comment (etsraphael) [inline src/index.html]: Useless space here. Please run `npx prettier --write .` to format the code

File: src/app/app.module.ts (hunk 1 above)
Comment (Nathanael-Rayapin) [reply src/app/app.module.ts]: I created a module file that is in charge of grouping all the Angular Material modules, I just didn't know where to put it so I put it in the root directory

File: src/index.html (hunk 2 above)
Comment (Nathanael-Rayapin) [reply src/index.html]: Done !

File: src/app/app.module.ts (hunk 1 above)
Comment (etsraphael) [reply src/app/app.module.ts]: I saw the update it's perfect!

**Pull request merged**